dask_jobqueue/deploy/cluster_manager.py

```diff
@@ -200,6 +212,7 @@ def scale(self, n=None, cores=None, memory=None):
         # TODO we should not rely on scheduler loop here, self should have its
         # own loop
         self.scheduler.loop.add_callback(self._scale, n, cores, memory)
+        self.wait_until_n_workers(wait_until_n)
```

Comment:
I think that having a keyword like this in many methods is a sign that maybe we shouldn't include it, and instead we should expect users to make two calls.

```diff
-cluster.scale(10, wait_until_n=10)
+cluster.scale(10)
+cluster.wait_until_n_workers(10)
```

Reply:
Otherwise keyword arguments like that tend to propagate to many new methods, resulting in an `n x m` maintenance problem.